SpatialOS/scripts/download_dependencies.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+cd "$(dirname "$0")"
+
+SDK_VERSION="$1"
+TOOLS_DIR="$(pwd)/../tools/${SDK_VERSION}"
+mkdir -p "${TOOLS_DIR}"
+
+retrievePackage() {
+  TYPE=$1
+  PACKAGE=$2
+  TARGETDIR=$3
+
+  pushd "${TOOLS_DIR}"
+  if [ ! -d "${TARGETDIR}" ]; then
+    spatial package get --force --unzip "${TYPE}" "${PACKAGE}" "${SDK_VERSION}" "${TARGETDIR}"
+  fi
+  popd
+}
+
+function isLinux() {
```

Comment:
It might make sense to move the code for determining your platform into a common util script to reduce code duplication.

Reply:
moving